examples/Python/Animation/Animation.pyde

```diff
@@ -0,0 +1,34 @@
+"""
+Animation
+https://rosettacode.org/wiki/Animation#Processing
+Processing 3.4
+2019-08 Jeremy Douglass
+2020-03 Alexandre Villares (Python Mode)
+Task: Create a window containing the string "Hello World! "
```

Comment:
Minor note -- in my merge I am now changing the spacing on all the tasks to match the Java examples (and the wiki style). Compact is nice too, I just think it is easier if everything is the same way everywhere.

These aren't requests on your PR -- just notes on my changes.